### DISCUSSION TITLE
Add PeerTube Liquid Tags

### DISCUSSION THREAD

**Discussion by siberkolosis** [forem/forem#16240] [Completed]
It would be great if forem could embed peertube, even use embed code from peertube is not work, PeertTube is the free and decentralized alternative to video platforms, providing over 400,000 videos published by 60,000 users and viewed over 15 million times

**Comment by obay2786** [comment]
I need also for this one liquid

**Comment by JennieOcken** [comment]
@siberkolosis and @obay2786 Thanks for the suggestion.

We have actually launched our unified embed approach.  This new approach actually will do the preview option for any URL with Open Graph or Twitter Card on it.  Try it out by using the { % embed URL } structure.  More details here: https://forem.dev/foremteam/changelog-a-new-and-streamlined-way-to-embed-rich-content-4h58

As such, I am going to actually gonna move any liquid tag requests to completed.  Happy embedding!
